<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>localization</class>
 <widget class="QWidget" name="localization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VColCOSP-VirtualPheromonePanel</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(19, 126, 124);</string>
  </property>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>VColCOSP - Localization</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>133</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_socket_ip">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Define robot &lt;span style=&quot; font-weight:600;&quot;&gt;ID&lt;/span&gt; with &lt;span style=&quot; font-weight:600;&quot;&gt;Pheromone Type&lt;/span&gt; (Color Channel) &lt;span style=&quot; font-style:italic;&quot;&gt;e.g.: 0:red;1,2:green;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(19, 126, 124);
color:rgb(255, 255, 127)</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;192.168.3.100&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>173</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>PORT</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_socket_port">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Define robot &lt;span style=&quot; font-weight:600;&quot;&gt;ID&lt;/span&gt; with &lt;span style=&quot; font-weight:600;&quot;&gt;Pheromone Type&lt;/span&gt; (Color Channel) &lt;span style=&quot; font-style:italic;&quot;&gt;e.g.: 0:red;1,2:green;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(19, 126, 124);
color:rgb(255, 255, 127)</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;6666&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_read_data">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>130</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton::disabled {
    border: 2px solid rgb(149, 149, 149);
	color:rgb(149, 149, 149);
	border-radius:10px;
}
QPushButton{
border: 2px solid rgb(19, 126, 124);
border-radius:10px;
}
</string>
   </property>
   <property name="text">
    <string>ReadData</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>| TCP Connect to LOCALIZATION</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_disconnect_to_loc">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton::disabled {
    border: 2px solid rgb(149, 149, 149);
	color:rgb(149, 149, 149);
	border-radius:10px;
}
QPushButton{
border: 2px solid rgb(19, 126, 124);
border-radius:10px;
}
</string>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_connect_to_loc">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton::disabled {
    border: 2px solid rgb(149, 149, 149);
	color:rgb(149, 149, 149);
	border-radius:10px;
}
QPushButton{
border: 2px solid rgb(19, 126, 124);
border-radius:10px;
}
</string>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_localization_dislay">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>242</y>
     <width>591</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(19, 126, 124);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
